code/game/objects/effects/decals/Cleanable/tracks.dm

```diff
@@ -199,7 +201,8 @@ var/global/list/image/fluidtrack_cache=list()
 			state=going_state
 			truedir=truedir>>4
 		var/icon/add = icon('icons/effects/fluidtracks.dmi', state, truedir)
-		add.SwapColor("#FFFFFF",track.basecolor)
```

Comment:
It appears a different color because of this change. 

These few lines also apparently adds the same icon as an overlay over the initial one?